core/main.py

```diff
@@ -2,7 +2,6 @@
 import thread
 import traceback
 
-
```

Comment:
This changes makes the code non-pep8 compliant

Reply:
No it doesn't. pycodestyle doesn't complain, and this change was done by my `isort` autofixer.

Also, this file isn't pep8 compliant already. I'd be happy to fix that up, but it seemed like unnecessary churn when it would be better to focus on getting the py3 pr merged.

Reply:
Okay!  Sounds good.